XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs

```diff
@@ -17,9 +17,27 @@ internal static class OculusPackageInstaller
 
         static OculusPackageInstaller()
         {
-            if (!EditorPreferences.Get($"{nameof(OculusPackageInstaller)}", false))
+            EditorApplication.delayCall += CheckPackage;
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Packages/Install Oculus Package Assets...", true)]
+        private static bool ImportLuminPackageAssetsValidation()
+        {
+            return !Directory.Exists($"{DefaultPath}\\Profiles");
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Packages/Install Oculus Package Assets...")]
+        private static void ImportLuminPackageAssets()
```

Comment:
Lumin? Probably copy paste issue here

Reply:
I don't see Lumin here...

Reply:
It's saying `ImportLuminPackageAssets()`

Reply:
👀 good catch lol